node/CHANGELOG.md

```diff
@@ -12,28 +12,25 @@ All notable changes to this project will be documented in this file.  The format
 
 ## [Unreleased]
 
-
-## [1.4.2] - 2021-11-11
-
-### Changed
-* There are now less false warnings/errors regarding dropped responders or closed channels during a shutdown, where they are expected and harmless.
-* Execution transforms are ordered by insertion order.
-
-### Removed
-* The config option `consensus.highway.unit_hashes_folder` has been removed.
-
-### Fixed
-* The block proposer component now retains pending deploys and transfers across a restart.
-
-
-## [1.4.0] - 2021-10-04
-
 ### Added
 * Add new event to the main SSE server stream accessed via `<IP:Port>/events/main` which emits hashes of expired deploys.
-* Add new event to the main SSE server stream across all endpoints `<IP:PORT>/events/*` which emits a shutdown event when the node shuts down.
+* Add new event to the main SSE server stream across all endpoints `<IP:Port>/events/*` which emits a shutdown event when the node shuts down.
```

Comment:
Ah, is that only on `feat-fast-sync` and not on `dev`?

Reply:
It is on `dev` as well, in the `[Unreleased]` section.
https://github.com/casper-network/casper-node/blob/dev/node/CHANGELOG.md

Reply:
Ah, makes sense, thanks!